etc/desimodel_sync_kpno_cron.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Exit on error
+set -e
+
+# Path to the sync directory, containing the desimodel data trunk
+# checkout and logs.
+syncdir=/n/home/datasystems/desimodel_sync
+logdir="${syncdir}/logs"
+svntrunk="${syncdir}/desimodel_trunk"
+
+logdate="$(date -u --iso-8601=seconds)"
+logname="sync_${logdate}"
+logfile="${logdir}/${logname}"
+
+echo "Running at ${logdate}" > "${logfile}"
+
+# Get the latest desiconda install
+desiconda=$(ls -d /software/datasystems/desiconda/20* | sort | tail -n 1)
+
+# Get the latest KPNO version of desimodules
+desimodules=$(ls ${desiconda}/modulefiles/desimodules/*-kpno | sort | tail -n 1 | xargs basename)
```

Comment:
For consideration: auto-deriving latest desiconda + latest desimodules is impressive but probably at least as fragile as maintaining hardcoded versions to use.  e.g. the existence of "21.3-kpno" instead of "21.3" was due to a post-facto realization that the 21.3 tag only worked at NERSC and not KPNO requiring a minor update.  Our intention is to make "21.5" work at both, thus requiring an update here to remove the -kpno suffix...

At the same time, since `desi_sync_focalplane` should be fairly stable, it shouldn't matter if this cronjob is running from a few versions behind, right?  What matters is that it is updating desimodel/trunk and committing it.   Although keeping it in sync is with the latest release is good, it's only critical to update the software release version used if there was some changes to `desi_sync_focalplane` itself (e.g. to accomodate a format change of the db dump).  (?)

Reply:
There are two options I can think of:

1.  Just agree on the naming convention and assume that.  We are logging the version that gets loaded.  This has the benefit that a new software tag just gets used on the next sync with no edits needed on the script or the crontab.
2.  Hardcode the version in the script.  In this case I would keep a checkout of desimodel master in the sync directory that is only used to access the script in desimodel/etc.  So to change the hardcoded version one would push the change and git pull in that checkout.

My fear is that this kind of thing can slip through the cracks, which is why I vote for option (1).

Reply:
OK, let's leave the code as is (including the "-kpno" hook suffix that is currently needed), and the update this as needed if/when the naming scheme changes (e.g. hopefully soon to make it even more standardized/maintainable).  If we find ourselves tripping on that, we can fall back to hardcoding the version to use (and thus experiment with a different set of failure modes...)